tests/test_modeling_physical_validation.py

```diff
@@ -0,0 +1,161 @@
+"""
+Tests for modeling
+"""
+
+import os
+import ast
+import astropy
+from numpy import testing as tst
+import numpy as np
+import clmm
+from clmm.constants import Constants as clmmconst
+
+# Read test case
+CODE_PATH = '/'.join(os.path.abspath(__file__).split('/')[:-2])
+VAL_FILES_PATH = '%s/tests/physical_values_nc'%CODE_PATH
+with open('%s/float_vals.txt'%VAL_FILES_PATH, 'r') as f:
+    TEST_CASE = ast.literal_eval(f.read())
+f.close()
+NC_PROF = np.genfromtxt('%s/numcosmo_profiles.txt'%VAL_FILES_PATH,
+                        names=True)
+NC_DIST = np.genfromtxt('%s/numcosmo_angular_diameter_distance.txt'%
+                        VAL_FILES_PATH, names=True)
+
+R3D = np.array(NC_PROF['r3d'])
+
+# Physical Constants
+G_PHYSCONST_CORRECTION = TEST_CASE['G[m3/km.s2]']/clmmconst.GNEWT.value
+CLMM_SIGMAC_PHYSICAL_CONSTANT = (clmmconst.CLIGHT_KMS.value*1000./clmmconst.PC_TO_METER.value)**2/\
+    (clmmconst.GNEWT.value*clmmconst.SOLAR_MASS.value/clmmconst.PC_TO_METER.value**3)
+TEST_CASE_SIGMAC_PHYSICAL_CONSTANT = (TEST_CASE['lightspeed[km/s]']*1000./TEST_CASE['pc_to_m'])**2/\
+    (TEST_CASE['G[m3/km.s2]']*TEST_CASE['Msun[kg]']/TEST_CASE['pc_to_m']**3)
+SIGMAC_PHYSCONST_CORRECTION = TEST_CASE_SIGMAC_PHYSICAL_CONSTANT/CLMM_SIGMAC_PHYSICAL_CONSTANT
+
+# Cosmology
+#COSMO_APY = astropy.cosmology.core.FlatLambdaCDM(H0=70, Om0=0.27, Ob0=0.045)
+COSMO_APY = astropy.cosmology.core.FlatLambdaCDM(H0=TEST_CASE['cosmo_H0'],
+                                                 Om0=TEST_CASE['cosmo_Om0'],
+                                                 Ob0=TEST_CASE['cosmo_Ob0'])
+COSMO_CCL = clmm.cclify_astropy_cosmo(COSMO_APY)
+
+# Sets of parameters to be used by multiple functions
+RHO_PARAMS = {
+    'mdelta':TEST_CASE['cluster_mass'],
+    'cdelta':TEST_CASE['cluster_concentration'],
+    'z_cl':TEST_CASE['z_cluster'],
+    'cosmo':COSMO_CCL,
+    }
+SIGMA_PARAMS = {
+    'mdelta':TEST_CASE['cluster_mass'],
+    'cdelta':TEST_CASE['cluster_concentration'],
+    'z_cl':TEST_CASE['z_cluster'],
+    'cosmo':COSMO_CCL,
+    'delta_mdef':TEST_CASE['mass_Delta'],
+    'halo_profile_model':TEST_CASE['density_profile_parametrization'],
+    }
+GAMMA_PARAMS = {
+    'mdelta':TEST_CASE['cluster_mass'],
+    'cdelta':TEST_CASE['cluster_concentration'],
+    'z_cluster':TEST_CASE['z_cluster'],
+    'z_source':TEST_CASE['z_source'],
+    'cosmo':COSMO_CCL,
+    'delta_mdef':TEST_CASE['mass_Delta'],
+    'halo_profile_model':TEST_CASE['density_profile_parametrization'],
+    'z_src_model':'single_plane',
+    }
```

Comment:
Move into function. We can just write one function that returns all of this stuff that each test calls.

Reply:
fixed